vmdb/spec/automation/unit/method_validation/orchestration_check_provisioned_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe "Orchestration check_provisioned Method Validation" do
+  STACK_EMS_REF = "12345"
```

Comment:
Can you move into a class or in a let?

a describe is not a class and this gets defined in global space.